ocular/src/chain/config.rs

```diff
@@ -11,3 +11,37 @@ pub struct ChainClientConfig {
     pub gas_adjustment: f64,
     pub gas_prices: String,
 }
+
+pub fn get_cosmoshub_config() -> ChainClientConfig {
+    ChainClientConfig {
+        chain_id: "cosmoshub-4".to_string(),
+        rpc_address: "https://cosmoshub-4.technofractal.com:443".to_string(),
+        grpc_address: "https://gprc.cosmoshub-4.technofractal.com:443".to_string(),
```

Comment:
It's possible that chains have a few rpc and grpc endpoints, right? Maybe when we auto populate configs like this with a vector, pulling all possible endpoints from the chain registry, and attempt to call healthy one's at runtime right before executing queries? This way we give the user more chances to have their query succeed? (vs if this single static endpoint is down for some reason we would just fail).